doc/installation.html

```diff
@@ -79,6 +79,21 @@ <h3>Directory structure</h3>
 would be replaced by <tt>core.so</tt>.  
 </p>
 
+<p>
+Alternatively, you can build and install LuaSocket using vcpkg dependency manager:
+
+<pre class=example>
+git clone https://github.com/Microsoft/vcpkg.git
+cd vcpkg
+./bootstrap-vcpkg.sh
+./vcpkg integrate install
```

Comment:
I'm fine with mentioning this can be installed via `vcpkg`. It looks like their package is even still current. However I'm not fine with including documentation about how to install `vcpkg`. If people don't know that they should be going to the source for that. Lets just stick to a basic mention of `vcpkg install luasocket`.

Reply:
Sure, thanks for your review!